backend/domain/Room.js

```diff
@@ -21,22 +21,10 @@ class Room {
         this.host = null;
     }
 
-    getRoomID() {
-        return this.roomID;
-    }
-
-    getGameState() {
-        return this.gameState;
-    }
-
     setGameState(gameState) {
```

Comment:
Can we remove this setter as well please?

Reply:
I've now removed it